frontend/tests/e2e/tests/eligibility/CoverageStatusCheckTest.js

```diff
@@ -166,6 +183,57 @@ test('Check properly filled form passes validation for an eligible PHN', async (
     .eql('')
 })
 
+test('Check invalid date does not overwrite previous entry', async (t) => {
+  const dateOfService = '20211201'
+  await t
+    // Given the page is filled out correctly
+    .typeText(CoverageStatusCheckPage.phnInput, '9347984074')
+    // and a valid date is entered
+    .typeText(CoverageStatusCheckPage.dateOfBirthInput, '19850915')
+    .pressKey('tab')
+    // entering an invalid date should cause it to be ignored and the previous value kept
+    .typeText(CoverageStatusCheckPage.dateOfBirthInput, '19850945')
+    .pressKey('tab')
+    .expect(CoverageStatusCheckPage.dateOfBirthInput.value)
+    .eql('19850915')
+    // The date input doesn't seem to take a new value without clearing the original value
```

Comment:
Need to include more tests for invalid date. Seems its throwing wrong error message and also not resetting DOB field on clicking clear button.

Reply:
Seems it was library issue. Updated package.